rest/src/main/kotlin/json/request/UserRequests.kt

```diff
@@ -1,31 +1,32 @@
 package dev.kord.rest.json.request
 
+import dev.kord.common.entity.Snowflake
 import dev.kord.common.entity.optional.Optional
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 @Serializable
 data class DMCreateRequest(
     @SerialName("recipient_id")
-    val userId: String
+    val userId: Snowflake,
 )
 
 @Serializable
 data class GroupDMCreateRequest(
     @SerialName("access_tokens")
     val tokens: List<String>,
-    val nick: Map<String, String>
+    val nick: Map<String, String>,
 )
 
 @Serializable
 data class CurrentUserModifyRequest(
     val username: Optional<String> = Optional.Missing(),
-    val avatar: Optional<String?> = Optional.Missing()
+    val avatar: Optional<String?> = Optional.Missing(),
 )
 
 @Serializable
 data class UserAddDMRequest(
     @SerialName("access_token")
     val token: String,
-    val nick: String
-)
\ No newline at end of file
+    val nick: String,
```

Comment:
!

Reply:
> What's p with all of these trailing `,`

My formatting settings (https://kotlinlang.org/docs/coding-conventions.html#trailing-commas) and me hitting Ctrl+Alt+L all the time, should I leave them out?